src/eventgenerator/generator/evaluator_test.go

```diff
@@ -213,15 +227,16 @@ var _ = Describe("Evaluator", func() {
 			}
 			evaluator.Start()
 			evaluator.Stop()
+			time.Sleep(1 * time.Second)
```

Comment:
No. I already showed you how to guarantee this. We should not be injecting random sleeps.